From 50dfada52cd84b226a7ca715c0ce5fd36cbe2870 Mon Sep 17 00:00:00 2001
From: Alison Wang <alison.wang@nxp.com>
Date: Fri, 21 Dec 2018 16:04:43 +0800
Subject: [PATCH 425/706] arm64: dts: ls1028ardb: Add sclk-strength property

This patch adds sclk-strength property for LS1028ARDB board. It will
change the SCLK pad drive strength to the maximum value.

Signed-off-by: Alison Wang <alison.wang@nxp.com>
(cherry picked from commit 036ca06cfc5db0b02c13cc33f6adc50b06c5ae3b)
Signed-off-by: Zumeng Chen <zumeng.chen@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-ls1028a-rdb.dts | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/boot/dts/freescale/fsl-ls1028a-rdb.dts b/arch/arm64/boot/dts/freescale/fsl-ls1028a-rdb.dts
index 0b9a6da9584d..bb672ab896d7 100755
--- a/arch/arm64/boot/dts/freescale/fsl-ls1028a-rdb.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-ls1028a-rdb.dts
@@ -139,6 +139,7 @@
 				VDDA-supply = <&reg_1p8v>;
 				VDDIO-supply = <&reg_1p8v>;
 				clocks = <&sys_mclk>;
+				sclk-strength = <3>;
 			};
 		};
 
-- 
2.17.1

